config/paths.test.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dmpSource": "src/digitalmarketplace/",
+  "dmpComponents": "src/digitalmarketplace/components/",
```

Comment:
This is `dmpSource` but the macro is `dmFooter`.

We should decide whether we want the abbreviation to be `dm` or `dmp`, and use that consistently everywhere.

Reply:
I think personally we should stay consistent with what we do with our Python libraries, which is `dm`

Reply:
Good catch, I agree we should use `dm`.